doc/setup.md

```diff
@@ -17,6 +17,8 @@ Issue reports and fixes are welcome.
 
 * aarch64 Linux with glibc 2.27+
 * aarch64 (Apple Silicon) macOS
+* Linux x86 (32-bit)
```

Comment:
```suggestion
* x86 (32-bit) Linux
```
I guess this would be more consistent. I assume we can't/don't want to make any glibc promises as it's from multilib and from a pinned Nix version